src/shell/if.go

```diff
@@ -22,3 +22,42 @@ func (i If) Ignore() bool {
 
 	return got == "true"
 }
+
+func (ifs Ifs) Ignore() bool {
+	for _, i := range ifs {
+		if If(i).Ignore() {
+			return true
+		}
+	}
+
+	return false
+}
+
+func convertToIfs(interfaces []interface{}) Ifs {
+	strs := make(Ifs, len(interfaces))
+	for index, value := range interfaces {
+		str, ok := value.(string)
+		if !ok {
+			continue
+		}
+		strs[index] = str
+	}
+	return strs
+}
+
+func checkIf(i any) bool {
```

Comment:
If `If` is a custom type instead of `interface{}`, this can also be a function on that type. Will make this cleaner.